cmd/allocator/main.go

```diff
@@ -173,26 +190,43 @@ func newServiceHandler(kubeClient kubernetes.Interface, agonesClient versioned.I
 	}
 	h.caCertPool = caCertPool
 
+	tlsCert, err := readTLSCert()
+	if err != nil {
+		logger.WithError(err).Fatal("could not load TLS certs.")
+	}
+	h.tlsCert = tlsCert
+
 	return &h
 }
 
+func readTLSCert() (*tls.Certificate, error) {
+	tlsCert, err := tls.LoadX509KeyPair(tlsDir+"tls.crt", tlsDir+"tls.key")
+	if err != nil {
+		logger.WithError(err).Infof("failed to generate credentials")
```

Comment:
Both places that call this function log the error, so no need to log it here.

Reply:
Removed it in (https://github.com/googleforgames/agones/pull/1638).